### PR TITLE
pip3: add page

### PR DESCRIPTION
**Pull request by volt4ire**
- [x ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pip3.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`pip3 install {{package_name}}`
+
+- Install a specific version of a package:
+
+`pip3 install {{package_name}}=={{package_version}}`
+
+- Upgrade a package:
+
+`pip3 install -U {{package_name}}`
```
Comment (sbrl) [inline pages/common/pip3.md]: Is there a long form of this option, and the ones below? In tldr pages we prefer to use the long-form options where available.

Comment (volt4ire) [reply pages/common/pip3.md]: Yep I just updated them accordingly in the new pull request

File: pages/common/pip3.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Upgrade a package:
+
+`pip3 install --upgrade {{package_name}}`
+
+- Uninstall a package:
+
+`pip3 uninstall {{package_name}}`
+
+- Save installed packages to file:
```
Comment (agnivade) [inline pages/common/pip3.md]: to _a_ file

File: pages/common/pip3.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`pip3 freeze > {{requirements.txt}}`
+
+- Install packages from file:
```
Comment (agnivade) [inline pages/common/pip3.md]: from _a_ file

File: pages/common/pip3.md (hunk 2 above)
Comment (mebeim) [reply pages/common/pip3.md]: I would rephrase this as:

    - Save the list of installed packages to a file:

Comment (volt4ire) [reply pages/common/pip3.md]: Thanks I just updated it incorporating your suggestions

Comment (sbrl) [reply pages/common/pip3.md]: Hey! I don't see the changes here. Maybe they didn't go through?

**Comment by mebeim** [comment]
Hey @volt4ire looks like you applied my suggestion... but to the wrong example. It should have been the one above that 👀

File: pages/common/pip3.md (hunk 2 above)
Comment (agnivade) [reply pages/common/pip3.md]: @volt4ire - You have updated the wrong line. :)

Comment (volt4ire) [reply pages/common/pip3.md]: Whoops sorry about that I just fixed it

**Comment by volt4ire** [comment]
> Nice, thank you!

Thanks for the awesome project - I'm happy to be able to contribute!
